src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx

```diff
@@ -61,13 +61,12 @@ class OrganizationIncidentsList extends AsyncComponent {
             {started.format('LL')}
             <LightDuration seconds={getDynamicText({value: duration, fixed: 1200})} />
           </div>
-
-          <div>
+          <AlignRight>
             <Count value={incident.uniqueUsers} />
-          </div>
-          <div>
+          </AlignRight>
+          <AlignRight>
             <Count value={incident.totalEvents} />
-          </div>
+          </AlignRight>
```

Comment:
Can we give this less of a presentational name and more semantic, e.g. `NumericColumn` or something